client/modules/i18n/currency.js

```diff
@@ -34,20 +63,19 @@ export function formatPriceString(formatPrice) {
     try {
       // we know the locale, but we don"t know exchange rate. In that case we
       // should return to default shop currency
-      if (typeof locale.currency.rate !== "number") {
+      if (!Match.test(userCurrency.rate, Number)) {
```

Comment:
Is there an advantage to using Match instead of typeof here?
Also, I think this requires adding `import { Match } from "meteor/check"` (to be on the safe side, as it's getting exported by Meteor -> but that's one of the meteor bad behaviors that we expect to see fixed)